src/components/Tabs/Tab/index.tsx

```diff
@@ -1,33 +1,37 @@
 import { motion } from 'framer-motion';
 
-interface TabItemProps<T> {
+export interface TabProps<T> {
   index: T;
-  label: string;
+  children: JSX.Element;
   activeTab: T;
   setActiveTab: (index: T) => void;
+  className?: string;
+  activeClassName?: string;
 }
 
-export function Tab<T>({ index, label, activeTab, setActiveTab }: TabItemProps<T>) {
+export function Tab<T>({ index, children, activeTab, setActiveTab, className, activeClassName }: TabProps<T>) {
   const getTabProps = (tabIndex: T) => ({
     'data-active': activeTab === tabIndex,
     onClick: () => setActiveTab(tabIndex),
   });
 
+  const defaultClassName = 'group tab relative w-full h-full p-2 ';
+
+  const combinedClassName = className || 'h-full text-lg font-bold text-primary py-5';
```

Comment:
I think this name is a bit misleading. I would expect it to be a combination of something while that's only done in l. 24. Maybe rename to `stylingClassName` or something?